### PR TITLE
Feature/create game

### PR DESCRIPTION
**Pull request by AzizaHelmy**

### CODE REVIEW

**Comment by AzizaHelmy** [comment]
NOTE: it not completed yet, but neccassry to review and merge it

File: app/src/main/java/com/honeycake/tictactoe/TicTacToeApplication.kt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.honeycake.tictactoe
+
+import android.app.Application
+import dagger.hilt.android.HiltAndroidApp
+
+/**
+ * Created by Aziza Helmy on 7/15/2023.
+ */
+@HiltAndroidApp
+class TicTacToeApplication : Application() {
```
Comment (TarekIdrees) [inline app/src/main/java/com/honeycake/tictactoe/TicTacToeApplication.kt]: remove this practice :D

File: app/src/main/java/com/honeycake/tictactoe/TicTacToeNavGraph.kt [hunk 2]
```diff
@@ -22,7 +22,7 @@ fun TicTacToeNavGraph() {
         }
         composable(Screens.CreateGame.route){
             CreateGameScreen(
```
Comment (TarekIdrees) [inline app/src/main/java/com/honeycake/tictactoe/TicTacToeNavGraph.kt:24]: Already changed to the recommended way from Bareq

File: app/src/main/java/com/honeycake/tictactoe/ui/composable/ButtonItem.kt [hunk 3]
```diff
@@ -21,6 +21,7 @@ import com.honeycake.tictactoe.ui.theme.Typography
 fun ButtonItem(
     modifier: Modifier = Modifier,
     text: String,
+    isEnabled: Boolean,
```
Comment (TarekIdrees) [inline app/src/main/java/com/honeycake/tictactoe/ui/composable/ButtonItem.kt]: make the default value true so you did not need to call isEnabled = true every line needed

**Review by TarekIdrees — CHANGES_REQUESTED** [review]
small changes :D 
Good Job 💯

**Review by TarekIdrees — APPROVED** [review]

**Pull request merged**